jupyterlab_git/handlers.py

```diff
@@ -871,6 +904,8 @@ def setup_handlers(web_app):
         ("/push", GitPushHandler),
         ("/remote/add", GitRemoteAddHandler),
         ("/remote/fetch", GitFetchHandler),
+        ("/remote/show", GitRemoteDetailsShowHandler),
+        ("/remote/remove", GitRemoteRemoveHandler),
```

Comment:
It will required a specific element to receive the name as parameter. So you should not place it in this list but add to `git_handlers` something like:

```py
git_handlers = # ... \
    + url_path_join(base_url, NAMESPACE + path_regex + r"/remote/%s" % r"(?P<name>[^/]*)" , GitRemoteRemoveHandler),
```

The handler argument is extracted from a named group in the regex.